clang/docs/ReleaseNotes.rst

```diff
@@ -955,6 +955,11 @@ Arm and AArch64 Support
     * Arm Neoverse-N3 (neoverse-n3).
     * Arm Neoverse-V3 (neoverse-v3).
     * Arm Neoverse-V3AE (neoverse-v3ae).
+- SVE and SVE2 have been moved to the default extensions list for ARMv9.0,
```

Comment:
I don't think this note to Clang is required, because the definition of `v9a` is an LLVM-internal concept.
From an end-user point of view, invoking Clang with `-march=armv9-a` should still enable SVE2.